_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1463,12 +1540,18 @@ The example code above does the following:
 . **Create IAM users**. For this example, we create `alice` and `bob`, and carol, adding `alice` to the `full-access`
   IAM group and `carol` to the `billing` IAM group. _Note_: your own IAM user (the one you created manually) should be
   in the `users` list; we'll use the `import` command to put this user under Terraform management shortly.
+# TODO: maybe the above works, to use `import` to put the user under Terraform management, but it was such a long frickin' section that I decided to skip the import. Instead, I used aws-vault to run the first terraform apply using the root account user, and then changed it to use the root IAM user once they were created by Terraform.
 
 . **Generate a password for each user**. We encrypt this password with that user’s PGP key from Keybase (we’ll come
   back to how to handle the passwords shortly).
 
+# TODO: Wow. This makes no sense at all. How the heck do I do this? Can't you just give me the code I'm supposed to add and where it goes?
+Luckily I read through the "testing" links that mentioned a backend.hcl... But we really shouldn't be giving that as advice here because
+attempting to test really screwed me up. Who knows how many accounts I'm going to end up with in this organization. I kinda want to start over
+with a whole new organization.
```

Comment:
I think the disconnect here is that users are really supposed to have read https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#deploy_using_terragrunt , but our callouts are not sufficient since it only references the entire guide (https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/#pre_requisites), which is too long for anyone to read carefully.

Reply:
Ah, yea, very good point on making this clearer... And also that because you can't automatically delete child AWS accounts, testing is actually very hard.